pkg/rpcclient/wsclient_test.go

```diff
@@ -148,12 +152,20 @@ func TestWSClientUnsubscription(t *testing.T) {
 
 func TestWSClientEvents(t *testing.T) {
 	var ok bool
+	txJSON, _ := json.Marshal(&transaction.Transaction{
+		Signers: []transaction.Signer{
+			{Account: util.Uint160{}},
+		},
+		Script: []byte("someScript"),
```

Comment:
It's a test, but still, usually it's `[]byte{byte(opcode.PUSH1)}` or something like that. It would also be nice to have a valid transaction script in the documentation example.